vietnamese/vietnamese.md

```diff
@@ -32,14 +32,21 @@
 ## Machine translation
 
 ### English-Vietnamese translation
-* Dataset is from [The IWSLT 2015 Evaluation Campaign](http://workshop2015.iwslt.org/downloads/proceeding.pdf), also be obtained from [https://github.com/tensorflow/nmt](https://github.com/tensorflow/nmt): `tst2012` is used for development while `tst2013` is used for test. Scores are computed for single models.
+* Dataset is from [The IWSLT 2015 Evaluation Campaign](http://workshop2015.iwslt.org/downloads/proceeding.pdf), also be obtained from [https://github.com/tensorflow/nmt](https://github.com/tensorflow/nmt).
 
-#### English-to-Vietnamese
+##### English-to-Vietnamese & `tst2015` is used for test
 
 | Model           | BLEU  |  Paper | Code | 
 | ------------- | :-----:| --- | --- | 
 | BPE-Dropout (2019) | 33.3 | [BPE-Dropout: Simple and Effective Subword Regularization](https://arxiv.org/abs/1910.13267) | | 
 | Nguyen and Salazar (2019) | 32.8 | [Transformers without Tears: Improving the Normalization of Self-Attention](https://arxiv.org/abs/1910.05895) | [Official](https://github.com/tnq177/transformers_without_tears) | 
+| Stanford (2015) |26.4 | [Stanford Neural Machine Translation Systems for Spoken Language Domains](https://nlp.stanford.edu/pubs/luong-manning-iwslt15.pdf) | | 
+
+
+##### English-to-Vietnamese & `tst2013` is used for test
```

Comment:
```suggestion
`tst2013` is used for test
---
```

This ensures that the links do not break in web rendering. This helps readers navigate and discover your contributions

This simplifies the overall layout as well.